test/test_dynamic_shapes.py

```diff
@@ -1,5 +1,5 @@
 # Owner(s): ["oncall: jit"]
-
+# ruff: noqa: F841
```

Comment:
FYI @anijain2305 for this one and fake tensor below. It would be good to enable these when we have a minute